packages/jest-mock-apollo/src/test/index.test.tsx

```diff
@@ -40,6 +41,9 @@ describe('jest-mock-apollo', () => {
       context: {
         client: createGraphQLClient(),
       },
+      childContextTypes: {
+        client: PropTypes.object,
```

Comment:
this was required to get the context pushed into mounted components.

Reply:
@lemonmade do you have insight on this? Feels like we should probably abstract away the requirement to specify the `childContextTypes` when mounting.

Reply:
I'm surprised this didn't come up before. We usually create a custom mount function that abstracts this away (as in `mountWithAppContext` for web)

Reply:
i suspect our internal `react-apollo` was wrapping with child context for us previously.